I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 19, in script call
    call battle(boss = bob, enemies = bane)
  File "game/battle.rpy", line 222, in script
    python:
  File "game/battle.rpy", line 232, in <module>
    _return = attack + " " + target
TypeError: unsupported operand type(s) for +: 'int' and 'str'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 19, in script call
    call battle(boss = bob, enemies = bane)
  File "game/battle.rpy", line 222, in script
    python:
  File "C:\Users\Matt\Desktop\renpy-6.16.5-sdk\renpy\ast.py", line 756, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\Matt\Desktop\renpy-6.16.5-sdk\renpy\python.py", line 1382, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/battle.rpy", line 232, in <module>
    _return = attack + " " + target
TypeError: unsupported operand type(s) for +: 'int' and 'str'

Windows-7-6.1.7601-SP1
Ren'Py 6.17.7.521
House Guard 0.0
